classes/Cookie.php

```diff
@@ -552,6 +552,9 @@ public function getSession($sessionId)
         }
 
         if (isset($session) && Validate::isLoadedObject($session)) {
+            // Update session date_upd
+            $session->save();
```

Comment:
If you check the `update` method, that's what is done automatically with date_upd :thinking:

Reply:
Not exactly, when the partial update is used and update_field is not empty Then the date fields are automatically added to the list of fields to update (if they exist). But since you don't use partial update it's just a regular update here so the whole object is saved in a query